atm.js

```diff
@@ -1,3 +1,64 @@
 $(document).ready(function(){
-// Uh oh -- it's saying `$` is undefined! Something's missing from `index.html`...
-});
+
+  var value =0
+  var balance
+  // var depositBalance = parseInt (balance) + parseInt( $("#depo").val())
+  // var savingbalance = $(".savingbalance").html()
+
+  $(".deposit").click(function(){
+
```

Comment:
You might want to save the value of `$("#depo").val()` to a variable inside of this event listener, since you already do it with `balance`. Would make your code more uniform.